src/CosmosDB/CosmosDB/Restore/GetAzCosmosDBMongoDBRestorableResource.cs

```diff
@@ -0,0 +1,59 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.CosmosDB.Helpers;
+using Microsoft.Azure.Commands.CosmosDB.Models;
+using Microsoft.Azure.Management.CosmosDB.Models;
+using System;
+using System.Collections;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.CosmosDB
+{
+    [Cmdlet(VerbsCommon.Get, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "CosmosDBMongoDBRestorableResource", DefaultParameterSetName = NameParameterSet), OutputType(typeof(PSDatabaseToRestore))]
+    public class GetAzCosmosDBMongoDBRestorableResource : AzureCosmosDBCmdletBase
```

Comment:
>GetAzCosmosDBMongoDBRestorableResource [](start = 17, length = 38)

Just thinking out loud:  May be it is a good idea to have a single cmdlet GetAzCosmosDBMongoDBRestorableResource and an optional paramter called ResourceType, if the customer provides ResourceType as Database then we call the Get RestorableMongoDatabase, similar for Collection.

Reply:
I think we did not get the new cmdlets reviewed from PS team, can you please go ahead with that process ? Would be good to get that done.

---
In reply to: [561414121](https://github.com/Azure/azure-powershell/pull/13983#discussion_r561414121) [](ancestors = 561414121)